app/assets/stylesheets/stories.scss

```diff
@@ -83,6 +88,18 @@
     }
   }
 
+  .extra_info_preview {
+    grid-area: extra-preview;
+    .content {
```

Comment:
this css is duplicated with the `.content` inside `.story_preview`, we can refactor this a bit to use the same style definition